bastion-executor/src/load_balancer.rs

```diff
@@ -47,46 +51,102 @@ impl LoadBalancer {
     }
 }
 
+/// Maximum number of core supported by mordern computers.
+const MAX_CORE: usize = 40;
+
 ///
 /// Holding all statistics related to the run queue
 ///
 /// Contains:
-/// * Global run queue size
 /// * Mean level of processes in the run queues
 /// * SMP queue distributions
-#[derive(Clone, Debug)]
 pub struct Stats {
```

Comment:
Let the Debug sit, missing_debug_impls are enabled.

Reply:
Done